File: chatwoot/csharp/src/OSEG.ChatwootExamples/UpdateConversationExample.cs
using System;
using System.Collections.Generic;
using System.IO;
using System.Text.Json;

using Com.Chatwoot.Api;
using Com.Chatwoot.Client;
using Com.Chatwoot.Model;

namespace OSEG.ChatwootExamples;

public class UpdateConversationExample
{
    public static void Run()
    {
        var config = new Configuration();
        config.ApiKey.Add("api_access_token", "USER_API_KEY");
        // config.ApiKey.Add("api_access_token", "AGENT_BOT_API_KEY");

        var updateConversationRequest = new UpdateConversationRequest(
        );

        try
        {
            new ConversationsApi(config).UpdateConversation(
                accountId: 0,
                conversationId: 0,
                data: updateConversationRequest
            );
        }
        catch (ApiException e)
        {
            Console.WriteLine("Exception when calling ConversationsApi#UpdateConversation: " + e.Message);
            Console.WriteLine("Status Code: " + e.ErrorCode);
            Console.WriteLine(e.StackTrace);
        }
    }
}

File: chatwoot/php/src/UpdateConversationExample.php
<?php

namespace OSEG\ChatwootExamples;

require_once __DIR__ . '/../vendor/autoload.php';

use SplFileObject;
use Chatwoot;

$config = Chatwoot\Client\Configuration::getDefaultConfiguration();
$config->setApiKey("api_access_token", "USER_API_KEY");
// $config->setApiKey("api_access_token", "AGENT_BOT_API_KEY");

$update_conversation_request = (new Chatwoot\Client\Model\UpdateConversationRequest())
    ->setPriority(null)
    ->setSlaPolicyId(null);

try {
    (new Chatwoot\Client\Api\ConversationsApi(config: $config))->updateConversation(
        account_id: 0,
        conversation_id: 0,
        data: $update_conversation_request,
    );
} catch (Chatwoot\Client\ApiException $e) {
    echo "Exception when calling ConversationsApi#updateConversation: {$e->getMessage()}";
}
